### PR TITLE
add flags to print item's real name or stat, and goblin alert

### PR DESCRIPTION
**Pull request by Zahariel1942**
add flag:  type, name, stat

### CODE REVIEW

File: D2Stats.au3 [hunk 1]
```diff
@@ -1230,6 +1246,7 @@ func OnClick_NotifyHelp()
 		'> 0-4 sacred - Item must be one of these tiers.', _
 		'   Tier 0 means untiered items (runes, amulets, etc).', _
 		'> normal superior rare set unique - Item must be one of these qualities.', _
+		'> type name stat - To print type name, real name or full stats.' _
```
Comment (kryshtof) [inline D2Stats.au3]: comma missing

**Comment by Zahariel1942** [comment]
any update?

**Comment by Zahariel1942** [comment]
LOL, someone stole the code and even signed it as their own.

https://imgur.com/3VcbSuU
https://imgur.com/AI7eWoy

**Comment by skyair00** [comment]
thanks

**Comment by tnt1094** [comment]
it can be fixed? it displays the name of items completely where unnecessary and not specified flag name, also it can not work correctly with color flags for example umo red but takes and displays the real color

**Pull request closed without merging**